build.gradle.kts

```diff
@@ -1,6 +1,8 @@
+import com.vanniktech.maven.publish.SonatypeHost
+
 plugins {
     kotlin("jvm") version "1.9.24"
-    `maven-publish`
+    alias(libs.plugins.maven.publishing)
```

Comment:
Decided to use the same plugin here since switching to sonatype requires a lot more finicky work to get right. Much cleaner just to use the plugin and keep consistent with other repos

Reply:
Let's keep the `maven-publish` related stuff for now since jitpack might need it. We'll delete it afterwards once we publish to maven.